tests/integration/targets/synchronize/tasks/main.yml

```diff
@@ -227,3 +227,36 @@
   - directory_a/foo.txt
   - directory_a
   - directory_b
+- name: synchronize files with quiet option
+  synchronize:
+    src: '{{output_dir}}/foo.txt'
+    dest: '{{output_dir}}/foo.result'
+    quiet: true 
+  register: sync_result
+  ignore_errors: true
+- assert:
+    that:
+    - '''files/directories have been synchronized'' in sync_result.msg'
+- name: Cleanup
+  file:
+    state: absent
+    path: '{{output_dir}}/{{item}}'
+  with_items:
+  - foo.result
+  - bar.result
+- name: synchronize files without quiet option 
+  synchronize:
+    src: '{{output_dir}}/foo.txt'
+    dest: '{{output_dir}}/foo.result'
+  register: sync_result
+  ignore_errors: true
+- assert:
+    that:
+    - '''files/directories have been synchronized'' not in sync_result.msg'
```

Comment:
This doesn't really make sense for me: despite this assertion, we have no certainty that this string is not in the result `msg` when the `synchronize` module succeeds, because we have no certainty that the `synchronize` module has succeeded, from a *logic* point of view.

Unless one *wants* to trigger an error, for example to test error handling of the module and ensure it behaves correctly under stress or with bad parameter values - and in that case we also *want* the playbook to not fail if a given task **fails as expected** - please don't use `ignore_errors=true`. Never. This makes just the tests passing even if the module is broken, that is ***exactly*** what the tests are made to avoid. An example of consistent use of `ignore_errors`: https://github.com/ansible-collections/community.general/blob/main/tests/integration/targets/filesize/tasks/errors.yml